sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1313,131 +1313,178 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the cosine inverse of the given value; the returned angle is in the range
-   * 0.0 through pi.
+   * @param e the value whose arc cosine is to be returned
+   * @return  cosine inverse of the given value in the range of 0.0 through pi,
+   *          as if computed by [[java.lang.Math#acos]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def acos(e: Column): Column = withExpr { Acos(e.expr) }
 
   /**
-   * Computes the cosine inverse of the given column; the returned angle is in the range
-   * 0.0 through pi.
+   * @param colName the value whose arc cosine is to be returned
+   * @return        cosine inverse of the given value in the range of 0.0 through pi,
+   *                as if computed by [[java.lang.Math#acos]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
-  def acos(columnName: String): Column = acos(Column(columnName))
+  def acos(colName: String): Column = acos(Column(colName))
```

Comment:
Why change columnName -> colName? doesn't seem to matter, so I'd avoid changing it if so.

Reply:
columnName was too long and it ran into the description in the generated doc.

Reply:
I don't think we should change the name for that reason ..